MAUI/Cartesian-Charts/Trackball.md

```diff
@@ -220,6 +222,206 @@ this.Content = chart;
 
 {% endtabs %}
 
+### Trackball Label Template
+
+You can customize the appearance of the Trackball label with your own template by using [TrackballLabelTemplate]() property of [CartesianSeries](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.CartesianSeries.html).
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCartesianChart>
+    <chart:SfCartesianChart.Resources>
+        <DataTemplate x:Key="trackballLabelTemplate">
+            <HorizontalStackLayout Spacing="5">
+                <Image Source="image.png" WidthRequest="20" HeightRequest="20"/>
+                <Label Text="{Binding Series.Label}" FontSize="15"  HorizontalOptions="Center"  TextColor="White"/>
+                <Label Text=" : " FontSize="15"  HorizontalOptions="Center" TextColor="White" />
+                <Label Text="{Binding Label,StringFormat='{0}M'}" FontSize="15" HorizontalOptions="Center" TextColor="White" />
+            </HorizontalStackLayout>         
+        </DataTemplate>
+    </chart:SfCartesianChart.Resources>
+
+   <chart:SfCartesianChart.TrackballBehavior>
+      <chart:ChartTrackballBehavior/>
+   </chart:SfCartesianChart.TrackballBehavior>
+    . . .
+    <chart:LineSeries ItemsSource="{Binding Data1}" 
```

Comment:
@EzhilarasanElangovan31  Label property not assigned in series

Reply:
Added label property in series